index.d.ts

```diff
@@ -230,6 +230,23 @@ declare module '@primer/components' {
   export const Sticky: React.FunctionComponent<PositionComponentProps>
   export const Fixed: React.FunctionComponent<PositionComponentProps>
 
+  export interface SideNavProps extends CommonProps, BorderProps, Omit<React.HTMLAttributes<HTMLDivElement>, 'color'> {
+    bordered?: boolean
+    variant?: 'normal' | 'lightweight'
+  }
+
+  export interface SideNavLinkProps extends CommonProps, TypographyProps, Omit<React.HTMLAttributes<HTMLAnchorElement>, 'color'> {
+    href?: string
```

Comment:
`href` isn't needed here since it's already part of the `HTMLAnchorElement` interface